lib/generators/driver/driver_generator.rb

```diff
@@ -12,8 +12,13 @@ def create_driver_dir_structure
     create_file "drivers/#{file_name}/lib/tasks/.keep", ''
     create_file "drivers/#{file_name}/extensions/.keep", ''
 
+    create_templated_files
+  end
+
+  def create_templated_files
     template 'routes.rb.erb',      "drivers/#{file_name}/config/routes.rb"
     template 'initializer.rb.erb', "drivers/#{file_name}/config/initializers/#{file_name}_feature.rb"
     template 'module.rb.erb',      "drivers/#{file_name}/app/models/#{file_name}.rb"
+    template 'README.md', "drivers/#{file_name}/README.md"
```

Comment:
Should we indent the second argument to line up with the rest of the lines in this method?

Reply:
yes, we should 👍